src/custom/components/Tooltip/TooltipMod.tsx

```diff
@@ -65,7 +65,7 @@ export function MouseoverTooltipContent({
   return (
     <TooltipContent {...rest} show={show} content={disableHover ? null : content}>
       <div
-        style={{ display: 'inline-block', lineHeight: 0, padding: '0.25rem' }}
+        // style={{ display: 'inline-block', lineHeight: 0, padding: '0.25rem' }}
```

Comment:
@elena-zh Disabled this inline style as it prevented us from properly styling things (without using !important). But this will have side-effects everywhere (tooltip padding) so we need to check carefully this PR.

Reply:
e.g. here is a side effect, which I will address in this case:

<img width="305" alt="Screenshot 2022-12-15 at 15 18 32" src="https://user-images.githubusercontent.com/31534717/207898129-c2de0b4c-c020-4c13-95d2-bb909ebce947.png">

Reply:
Fixed this